activities/Measure.activity/locale.ini

```diff
@@ -144,4 +168,11 @@ PastWeek=Semana passada
 PastMonth=Mês passado
 PastYear=Ano passado
 UserJoined={{name}} ingressou
-UserLeft={{name}} saiu
\ No newline at end of file
+UserLeft={{name}} saiu
```

Comment:
Do not try to localize in Portuguese. Just let strings in English. It's better to let native language speaker do localization.